package.json

```diff
@@ -5,6 +5,10 @@
   "license": "MIT",
   "author": "Bouke van der Bijl <bouke@shopify.com>",
   "description": "A CLI for converting Shopify’s CoffeeScript to ESNext.",
+  "scripts": {
+    "deploy": "npm run lint && node ./script/deploy.js",
```

Comment:
If there's no directory for scripts already, can we name it `scripts`? Plural makes more sense to my brain :D